core/crypto/rng_hw.c

```diff
@@ -7,12 +7,25 @@
 #include <tee/tee_cryp_utl.h>
 #include <types_ext.h>
 
+TEE_Result __weak hw_get_random_bytes(void *buf, size_t blen)
```

Comment:
I think it is not nice to have 2 functions name that identical
`hw_get_random_byte()`
`hw_get_random_bytes()`

Reply:
Which is why I removed the singular version by the end of the series, so there is only one of these functions.

Reply:
ok